server/service/src/sync/translations/requisition.rs

```diff
@@ -382,7 +382,7 @@ fn from_legacy_status(
             _ => return None,
         },
         LegacyRequisitionType::Response => match status {
-            LegacyRequisitionStatus::Sg => return None,
+            LegacyRequisitionStatus::Sg => RequisitionRowStatus::New,
```

Comment:
returning `None` meant that the fn which is mapping the status raises an error. This prevents processing of requisitions.